geopandas/io/file.py

```diff
@@ -128,13 +132,14 @@ def infer_schema(df):
     except ImportError:
         from ordereddict import OrderedDict
```

Comment:
Btw, has anyone an idea what is this? Some remains of old python? I haven't found what `ordereddict` module is supposed to be.

Reply:
Maybe an old backport module for older python versions or something? 
In any case, with python >= 3.5, OrderedDict is always importable, so this try/except can be removed.